src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -74,27 +75,38 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String email = request.getParameter(EMAIL_PARAMETER_KEY);
 
     if (email != null) {
```

Comment:
Can you add a comment to describe why you are searching for email and username params before sending the self user profile? Just so others are aware.

Reply:
I added comments for both options of username and email in last commit.